iron-collapse.html

```diff
@@ -216,6 +216,9 @@
       // Set the final size.
       if (size === 'auto') {
         this.style[this._dimensionMax] = '';
+      // call _transitionEnd if no transition is performed, styles need to be set
+      } else if (size === startSize) {
+        this._transitionEnd();
       } else {
         this.style[this._dimensionMax] = size;
       }
```

Comment:
I'd suggest we take care of calling `_transitionEnd` when needed in `updateSize`:
```javascript
this._updateTransition(false);
var willAnimate = animated && !this.noAnimation && this._isDisplayed;
// If we can animate, must do some prep work.
if (willAnimate) {
  var startSize = this._calcSize();
  // For `auto` we must calculate what is the final size for the animation.
  // After the transition is done, _transitionEnd will set the size back to `auto`.
  if (size === 'auto') {
    this.style[this._dimensionMax] = '';
    size = this._calcSize();
  }
  // Go to startSize without animation.
  this.style[this._dimensionMax] = startSize;
  // Force layout to ensure transition will go. Set scrollTop to itself
  // so that compilers won't remove it.
  this.scrollTop = this.scrollTop;
  // Enable animation.
  this._updateTransition(true);
  // If final size is the same as startSize it will not animate.
  willAnimate = (size !== startSize);
}
// Set the final size.
this.style[this._dimensionMax] = (size === 'auto') ? '' : size;
// If it won't animate, call transitionEnd to set correct classes.
if (!willAnimate) {
  this._transitionEnd();
}
```
This will allow you to remove the check for `noAnimation` in `_openedChanged` https://github.com/PolymerElements/iron-collapse/pull/63/files#diff-9c9f96df8dd507160ccdd7e52f160ed3R262